README.adoc

```diff
@@ -293,6 +293,42 @@ render it as:
 image::{{ image_base_path }}/{{ diagram.xmi_id }}.{{ format | default: 'png' }}[]
 -----
 
+=== Rendering a LutaML GML Dictionary: `lutaml_gml_dictionary`
+
+This command allows to render a LutaML GML Dictionary.
+
+[source,adoc]
+-----
+lutaml_gml_dictionary::[xml_path="/path/to/dictionary.xml",source="source_string",template_path="/path/to/template.liquid"]
```

Comment:
Please support these two syntaxes for the user document:

The first method allows the user to write the liquid template inline.
```adoc
[lutaml_gml_dictionary,"iur/codelists/3.1/BuildingInstallation_class.xml",root]
----
== GML Dictionary for {{ filename }}

{% render 'gml_dict_template', root: root, filename: filename %}
----
```

The second method allows the user 
```adoc
lutaml_gml_dictionary::["iur/codelists/3.1/BuildingInstallation_class.xml",context=root,template="/path/to/template.liquid"]
```

Is it difficult to support?

Reply:
I will investigate this.